src/jinja2_fragments/__init__.py

```diff
@@ -85,3 +85,57 @@ def render_block(
         return environment.concat(block_render_func(ctx))  # type: ignore
     except Exception:
         environment.handle_exception()
+
+
+def render_block_list(
+    environment: Environment,
+    template_name: str,
+    block_name_list: list[str],
```

Comment:
As the previous comment, I think `block_names` is cleaner, as we already have the type annotations to indicate that it's expecting a list.